src/redux/reducers/authentication.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { authentication } from "./authentication";
+
+describe("authentication", () => {
+  it("should", () => {
```

Comment:
you probably forgot to add a description

Reply:
it should!